qiita_db/support_files/patches/python_patches/14.py

```diff
@@ -0,0 +1,15 @@
+from qiita_db.sql_connection import SQLConnectionHandler
+from os.path import basename
+
+# This changes all analysis files to be relative path instead of absolute
```

Comment:
Add date and move all comments to the first line of the file (for consistency with other patches)